.env-dist

```diff
@@ -72,7 +72,9 @@ HIBP_NOTIFY_TOKEN=unsafe-default-token-for-dev
 HIBP_BREACH_DOMAIN_BLOCKLIST=a-blocked-domain.com,another-blocked-domain.org
 
 # OneRep API for exposure scanning
+ONEREP_API_BASE=https://api.onerep.com
 ONEREP_API_KEY=
+ONEREP_WEBHOOK_SECRET=
```

Comment:
Is this different from ONEREP_API_KEY?

Reply:
Ah yes, that makes sense. We need to register all 3 endpoints, ask SRE to add the env vars for stage/prod, and record the dev secrets in our shared space (also add the env var to heroku)